scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -134,7 +135,7 @@ object ScroogeSBT extends Plugin {
     // returns Seq[File] - directories that include thrift files
     scroogeUnpackDeps <<= (
       streams,
-      configuration,
+      configuration in Compile,
```

Comment:
This has to do with sbt scopes. I sort of forgot some of the intricacies here, but I remember that before this change it was not possible for a user to change setting keys in the build file because the setting that they changed was not in the same scope as the one being set and read by the plugin.

This is also a somewhat orthogonal change. My understanding is that currently, it is not possible for a user to change some of the important `SettingKeys`, if this is not the case, then I could be missing something.